[docs] Cloud Reorganize Deploy (#10706)
yugabyte/yugabyte-db: docs/content/latest/yugabyte-cloud/cloud-connect/_index.md

---
title: Connect to clusters in Yugabyte Cloud
linkTitle: Connect to clusters
description: Connect to clusters in Yugabyte Cloud.

> headcontent: Connect to your cluster using the cloud shell, a client shell, and from applications

Reviewer: Add period

---
title: Connect to clusters in Yugabyte Cloud
linkTitle: Connect to clusters
description: Connect to clusters in Yugabyte Cloud.
headcontent: Connect to your cluster using the cloud shell, a client shell, and from applications.
image: /images/section_icons/index/quick_start.png
aliases:
  - /latest/deploy/yugabyte-cloud/connect-to-clusters/
  - /latest/yugabyte-cloud/connect-to-clusters/
  - /latest/yugabyte-cloud/cloud-basics/connect-to-clusters/
menu:
  latest:
    identifier: cloud-connect
    parent: yugabyte-cloud
    weight: 30
isTocNested: true
showAsideToc: true
---

Connect to clusters in Yugabyte Cloud in the following ways:

- From a browser - Use cloud shell to connect to your database using any modern browser. No need to set up an IP allow list, all you need is your database password.
- From your desktop - Install the ysqlsh or ycqlsh client shells to connect to your database from your desktop. Your computer must be added to the cluster IP allow list and an SSL connection is required.
- Applications - Connect your applications to your cluster database.

> Once you have connected to the database, you can add database users so that other team members can access the database.

Reviewer: ```suggestion
Once you have connected to the database, you can add database users for other team members to access the database.
```
